next/13/replace-next-head/index.ts

```diff
@@ -1784,6 +1784,12 @@ export const repomod: Repomod<Dependencies, Record<string, unknown>> = {
 				String(options.metadata ?? '{}'),
 			);
 
+			if (Object.entries(metadata ?? {}).length === 0) {
```

Comment:
can you explain what was the problem?

Reply:
without this early return upsertData command will be pushed even if no metadata was inserted. And engine will produce a job that will be visible in extension codemod run results. 
<img width="921" alt="image" src="https://github.com/intuita-inc/codemod-registry/assets/125881252/5012da31-6216-4a71-8491-241ab837a668">